NuRadioReco/framework/base_trace.py

```diff
@@ -299,20 +299,26 @@ def deserialize(self, data_pkl):
         if 'trace_start_time' in data.keys():
             self.set_trace_start_time(data['trace_start_time'])
 
-    def add_to_trace(self, channel):
+    def add_to_trace(self, channel, min_residual_time_offset=1e-5, raise_error=True):
```

Comment:
unit missing. It should be `min_residual_time_offset=1e-5 * units.ns`

Reply:
... also in the docstring